### ISSUE TITLE
unable to use example for GCE as is

### ISSUE THREAD

**Issue by VikramTiwari** [jetstack/kube-lego#100]
The echoserver service is not getting an IP assigned to it. Not sure what is causing the issue.

**Comment by akoenig** [comment]
Yeah, I'm facing the same problem :( Additionally, the created LoadBalancer indicates unhealthy nodes. :/

**Comment by simonswine** [comment]
The IP is gonna get assigned to the ingress object for the GCE example or to the nginx service for the other.

To debug further this needs information++

**Comment by VikramTiwari** [comment]
@simonswine I can do this again. Let me know which logs will be helpful to debug this.
